includes/template-library/manager.php

```diff
@@ -450,9 +450,16 @@ public function direct_import_template() {
 	 * @return mixed Whether the export succeeded or failed.
 	 */
 	public function import_template( array $data ) {
+		// If the template is a JSON file, allow uploading it.
+		add_filter( 'elementor/allow-file-type/json', [ $this, 'enable_json_template_upload' ] );
+		add_filter( 'elementor/files/allow_unfiltered_upload', [ $this, 'enable_json_template_upload' ] );
```

Comment:
Remove this filter and handle deprecated methods

Reply:
The filter cannot be removed, as it is necessary for different usages in the product.
The specific file type filter is necessary for JSON files in order to enable importing single templates.
@kobizz

Reply:
Handling deprecated methods - done.